monasca_persister/repositories/influxdb/abstract_repository.py

```diff
@@ -56,8 +58,12 @@ def _write_batch(self, data_points, database):
                                                    database=database)
                 break
             except influxdb.exceptions.InfluxDBClientError as ex:
-                if (str(ex).startswith(DATABASE_NOT_FOUND_MSG) and
```

Comment:
I looks like `if 'DATABASE_NOT_FOUND_MSG' in ex.message and ...` should work from: https://github.com/influxdata/influxdb-python/blob/master/influxdb/exceptions.py

Reply:
As in, there is an expected format that the error comes back as: which is as a json string blob... so we attempt to parse it... and if it cant be parsed because it isn't in the expected format, escalate the exception.... only create the DB if the error matches our expected format.